recipe/patch_yaml/scipy.yaml

```
@@ -1,3 +1,19 @@
+# upper bound for numpy in version 1.9.0, 1.9.1, 1.9.2 not tight enough
+# which allowed installation next with numpy 1.26
+
+if:
+  name: scipy
+  version_le: 1.9.2
+  version_ge: 1.9.0
+  timestamp_lt: 1737471013000
+
+then:
+  - replace_depends:
+      old: numpy<2.0a0
+      new: numpy<1.26.0
```

```suggestion
      old: numpy >=1.19.5,<2.0a0
      new: numpy >=1.19.5,<1.26.0
```

Turns out, we need the full dependency line.